tool/forcefield/src/main/java/org/openscience/cdk/modeling/builder3d/MMFF94BasedAtomTypePattern.java

```diff
@@ -163,7 +163,7 @@ private void createPattern() {
         //cdk.bug 3515122 fixed
         atomTypePatterns.add(Pattern.compile("N-[1-3];[CHN]{1,3}.{1}+[A-Z]{0,3}+[,]?+=O[CNXO].*+"));
         //NC=0 amid
-        atomTypePatterns.add(Pattern.compile("N-1-2];[CH]{1}+=S[(].*+"));
+        atomTypePatterns.add(Pattern.compile("N-[1-2];[CH]{1}+=S[(].*+"));
```

Comment:
Maybe... but this class should be deprecated and has been replaced by [Mmff](http://cdk.github.io/cdk/2.0/docs/api/index.html). If it's not deprecated I'll get on it.